catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/tags/TagResource.java

```diff
@@ -93,7 +93,8 @@ public void initialize(CatalogApplicationConfig config) throws IOException {
         tagFile -> {
           try {
             LOG.info("Loading tag definitions from file {}", tagFile);
-            String tagJson = IOUtil.toString(getClass().getClassLoader().getResourceAsStream(tagFile));
+            String tagJson =
+                IOUtil.toString(Objects.requireNonNull(getClass().getClassLoader().getResourceAsStream(tagFile)));
             TagCategory tagCategory = JsonUtils.readValue(tagJson, TagCategory.class);
```

Comment:
TagRepository does not inherit from EntityRepository and the entity Tag is different from TagCategory.

Hence we cannot easily use the new initSeedData API in EntityRepository